Makefile

```diff
@@ -1,6 +1,6 @@
 #_*_MakeFile_*_
 CC = gcc 
-_CFLAGS = -Os -s -g -std=gnu99
+_CFLAGS = -O2 -s -g -std=gnu99
```

Comment:
This should be its own commit, with its own description explaining why this change was made. If this needed to be bisected by someone else, knowing why this change was made may make a big difference, which is why it shouldn't be lumped in with another change.